packages/cypress/src/takeSnapshot.ts

```diff
@@ -10,7 +10,7 @@ export const takeSnapshot = (
       resolve(null);
     }
 
-    const domSnapshot = snapshot(doc);
+    const domSnapshot = snapshot(doc, { recordCanvas: true });
```

Comment:
We'll also want this on the `commands.ts` file's `snapshot()` call (for manual snapshots). Maybe it's worth making those a common function now that we have options.

Reply:
@skitterm I'm not sure what you mean. `commands.ts` calls this `takeSnapshot` function, not the rrweb `snapshot` function directly. Am I missing something?